docs/source/reference/yaml-spec.rst

```diff
@@ -95,11 +95,12 @@ describe all fields available.
       # image_id: projects/deeplearning-platform-release/global/images/family/tf2-ent-2-1-cpu-ubuntu-2004
 
     file_mounts:
-      # Uses rsync to copy local files to all nodes of the cluster.
+      # Uses rsync to sync local files/directories to all nodes of the cluster.
       #
       # If symlinks are present, they are copied as symlinks, and their targets
       # must also be synced using file_mounts to ensure correctness.
-      /remote/path/datasets: /local/path/datasets
+      /remote/path/file: /local/path/file
```

Comment:
how about `/remote/path1/file: /local/path1/file` and `/remote/path2: /local/path2` to distinguish the folder path and the file path?

Reply:
Good point - Changed to:
```
      /remote/dir/file: /local/dir/file
      /remote/dir: /local/dir
```

Reply:
Ahh, I meant the prefix of the file may need to be different from the dir below? Otherwise, I am not sure if it will work correctly uploading the file to a dir first and then upload a folder to the same dir.

Reply:
Good catch! Tried it and we correctly erred out. Updated yaml, PTAL.